pkg/rpc/client.go

```diff
@@ -315,3 +319,43 @@ func (c *Client) GetHealth(ctx context.Context) (*string, error) {
 	}
 	return &resp.Result, nil
 }
+
+// GetIdentity Returns the identity pubkey for the current node
```

Comment:
Super minor thing here, but can we have the `"returns"` lowercase such that it reads like a sentence? Ditto for the other new methods. I believe that is the Go way to do it.

Reply:
done